.github/actions/deploy/action.yml

```diff
@@ -27,16 +27,27 @@ inputs:
   IMAGE_NAME:
     description: Name of the image
     required: true
+  KEY_VAULT:
+    description: Name of the Key Vault
+    required: true
+  CREATE_LEAD_ENDPOINT:
+    description: Lead endpoint
+    required: true
+  GOOGLE_RECAPTCHA_KEY_V2:
+    description: Google recaptcha key
+    required: true
+  GOOGLE_RECAPTCHA_SITE_KEY:
+    description: Google recaptcha site key
+    required: true
```

Comment:
I'm not a fan of these params given that this should only be set for PR slots - can we move this logic to a bicep file that sets up the slot with the permissions required?

Reply:
I don't get this, could you please explain it a bit more? either way we need to pass the params to the bicep files to access those from .env and some from GitHub secrets.